vllm/model_executor/models/minicpm.py

```diff
@@ -503,10 +500,8 @@ def __init__(
             self.model.make_empty_intermediate_tensors)
 
     def _init_model(self):
```

Comment:
Let's update this function to pass config and prefix so we can pass them through init method (we also should use maybe_prefix here)

Reply:
nice catch! fixed in [59a8c2d](https://github.com/vllm-project/vllm/pull/10205/commits/59a8c2d21b1344c821f4a7f9e019d835e192921d)